addon/components/kinetic-form.js

```diff
@@ -50,7 +50,15 @@ export default Component.extend({
       let validators = {};
       let properties = get(this, 'properties');
       for (let property of properties) {
-        validators[property.key] = validatorsFor(property);
+        // FIXME: probably better to do this in the `schemaParser` but this is quick fix to get schema props to validate when they are inside a section
+        if (property.type === 'section') {
```

Comment:
The JSON:Schema-form spec has more then just sections that use `items` It may be better to test for that instead of the specific type. Doing it this way means that adding another type will not gain the benefits of validations.

Might I suggest this:

```js
validators: computed('properties.@each.required', {
  get() {
    let validators = {};
    let properties = get(this, 'properties');

    function buildValidators(items = []) {
      for (let item of items) {
        if (item.items) { buildValidators(item.items); }
        if (!item.key) { continue; }
        validators[item.key] = validatorsFor(item);
      }
    }

    buildValidators(properties);
    return validators;
  }
}),
```